packages/terra-clinical-item-view/lib/ClinicalItemView.scss

```diff
@@ -0,0 +1,175 @@
+@import '~terra-mixins';
+@import './variables';
+@import './mixins';
+
+// Temporary until it is added to application.
+.terra-ClinicalItemView {
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  font-size: 87.5%;
```

Comment:
Should this use the font size and family variable declared in terra? 
https://github.com/cerner/terra-ui/blob/master/packages/terra-application/src/_variables.scss#L3
https://github.com/cerner/terra-ui/blob/master/packages/terra-legacy-theme/src/_typography.scss#L1